docs/htdocs/info/docs/tools/vep/script/vep_download.html

```diff
@@ -133,6 +133,8 @@ <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_R
 
     <li>New plugins (supported on CLI): </li>
       <ul>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/DeNovo.pm" rel="external">DeNovo</a> - identifies de novo variants in a VCF file. This plugin requires a pedigree (.ped) file.</li>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/SpliceVault.pm" rel="external">SpliceVault</a> - predict exon-skipping events and activated cryptic splice sites based on the most common mis-splicing events around a splice site</li>
```

Comment:
`predict` -> `predicts`? Similar to the 3rd-person verbs beginning the other lines?

Reply:
Could do for consistency I guess